src/file-1.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonarcloud-github-static-resources/v1/common/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:37429/organizations/org1/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:37429/project/issues?id=sample&pullRequest=7002&issues=AWlr8zAwM1H-wc5kn_1T&open=AWlr8zAwM1H-wc5kn_1T)

[ISSUE_KEY]: AWlr8zAwM1H-wc5kn_1T